### PR TITLE
Added support for course problems endpoint

### PR DESCRIPTION
**Pull request by clintonb**
The submission_counts method has been removed.

@dsjen @rocha @jbau @dylanrhodes

### CODE REVIEW

File: analyticsclient/module.py [hunk 1]
```diff
@@ -29,22 +28,6 @@ def answer_distribution(self, data_format=DF.JSON):
 
         return self.client.get(path, data_format=data_format)
 
-    def submission_counts(self, module_ids, data_format=DF.JSON):
```
Comment (rocha) [inline analyticsclient/module.py]: Maybe this deletion should be done in a separate commit? Also, is it possible that there is code that  depends on this method?

Comment (clintonb) [inline analyticsclient/module.py]: There should be nothing using this endpoint. It was added for the answer distribution work.

**Comment by rocha** [comment]
If possible, move the function deletion to a separate commit. Other than that, :+1:

File: analyticsclient/module.py (hunk 1 above)
Comment (rocha) [inline analyticsclient/module.py]: Yeah, it makes sense after reading the equivalent PR in the API. :+1:

Comment (clintonb) [inline analyticsclient/module.py]: Split the commit.
